Lib/Stack.hpp

```diff
@@ -124,6 +129,45 @@ class Stack
     std::swap(*this,s);
   }
 
+/**
```

Comment:
This function is cool! :)

Reply:
I agree! But...thinking more about it, won't the second stack (on destruction or resize) try to de-allocate a pointer halfway into an allocated region?

Reply:
A good point! 

Even on destruction, every list tries to deallocate.

And these will pass a wrong number of elements to `DEALLOC_KNOWN` because neither will know the original `_capacity`?

Reply:
> I agree! But...thinking more about it, won't the second stack (on destruction or resize) try to de-allocate a pointer halfway into an allocated region?

Yesss exactly! Sounds like this is not allowed then...? I wasn't aware of that. I think the function is not in use actually. I wrote it at some point coz i needed it and kept it coz it seems useful but we can remove it again I think.